modules/development/ssh.nix

```diff
@@ -31,6 +31,5 @@ in
     config = mkIf cfg.enable {
       services.openssh.enable = true;
       users.users.root.openssh.authorizedKeys.keys = authorizedKeys;
-      users.users.${config.ghaf.users.accounts.user}.openssh.authorizedKeys.keys = authorizedKeys;
```

Comment:
I understand this is considered controversial by some but just removing it without any documented placeholder mechanism (other than developers add their keys themselves somewhere and rebuild the image) currently breaks testing, debugging and development flows (incl. `nixos-rebuild switch`) for `-debug` image.

This will have a major impact to development productivity when the step must be done manually for every PR to ghaf. And people must remember to remove their hacked keys from their PRs not to cause extra effort for reviewers.

There, I said it.

Reply:
This needs to be changed to apply to release only, where ssh currently isn't included (and thus functionality does not work) at all. So maybe revert this and leave user keys in for now.

Perhaps it makes sense to e.g. name the "admin" to "ghaf". This will still allow older scripts to run with ghaf user, and potential sudo cmds as it is in wheel.

Reply:
Thanks @mbssrc - appreciate the common sense.

I re-requested comments on practical `-debug`-support replacement mechanism from other Ghaf contributors. So far the only proposals have been to
1. clear the `authorizedKeys` list with pre-defined public keys in the Ghaf `main`-branch and everyone rebuilds the `-debug`-images with their own keys
2. add some `--impure` hack to inject personal public key to `-debug`-builds which is essentially a variant of 1.

I think I concluded the outcome of 1. in my previous comment.

Reply:
I have to test that. I tested only `sudo su` and `login root` as `operator` from within the `-debug` in `ghaf-host`. Both failed. Even that limits the usage quite much.

Reply:
Please note for `-debug` builds,  `operator` is not part of wheel group. Try with `admin` user.

Reply:
Right - I think we better document your comments already early on under the Testing section of this PR description as this will have wide impact to automated tests, development practices, etc.- even if we leave the development/debug access public keys in place also for non-root users. What do you think?

Reply:
I have updated Testing Section. Leaving development/debug access public keys only for `-debug` builds is okay. @mbssrc are you okay with that?

Reply:
Added keys for `root` and `operator` users.